.github/workflows/deploy.yml

```diff
@@ -14,6 +14,7 @@ on:
         type: choice
         options:
           - review
+          - staging
```

Comment:
isn't this similar to https://github.com/DFE-Digital/early-careers-framework/pull/5798/files#r2093067771 and we don't need it?

Reply:
we have a manual deployment action here instead https://github.com/DFE-Digital/early-careers-framework/actions/workflows/deploy_to_environment.yml which includes staging